idea/src/org/jetbrains/kotlin/idea/inspections/conventionNameCalls/ReplaceCallWithBinaryOperatorInspection.kt

```diff
@@ -123,6 +123,7 @@ class ReplaceCallWithBinaryOperatorInspection : AbstractApplicabilityBasedInspec
         return when (identifier) {
             OperatorNameConventions.EQUALS -> {
                 if (!dotQualified.isAnyEquals()) return null
+                if (dotQualified.isFloatingPointNumberEquals()) return null
```

Comment:
Here we should not just drop a suggestion. It should be kept, but its highlight type should be changed to `INFORMATION`, and its fix text changed to `Replace total order equality with IEEE 754 equality`.

Reply:
(See related functions above)

Reply:
I fixed it.